styles.css

```diff
@@ -36,22 +36,21 @@ a {
     color: #ececec;
 }
 
+h1 {
+    font-size: 2em;
+}
+
 /*Navigation*/
-#nav_bar {
+nav {
     position: fixed;
     top: 0;
     width: 100%;
     height: 4rem;
     background-color: #385170;
     z-index: 2;
-    /*display: flex;
-    flex-wrap: wrap;
-    flex-grow: 1;
-    justify-content: space-between;
-    align-items: center;*/
 }
 
-#nav_bar ul {
+nav ul {

```

Comment:
This is an example of a time where we should prefer to use the more generic selector. In this case there is only one 'nav' element so it has the same effect but if we had another page with another nav bar, or something in javascript that changed our nav element for another one we *would* want the same styling to apply.